### PR TITLE
Add semconv-mobile-approvers

### PR DESCRIPTION
**Pull request by breedx-splk**
This adds two paths for the newly minted `@open-telemetry/semconv-mobile-approvers` group:

* `/docs/mobile/` - Where mobile specific conventions will be captured
* `/model/logs/mobile*` - For definitions of mobile events via yaml

Relates to #67, #343, and #348.

### CODE REVIEW

**Review by reyang — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -41,4 +41,8 @@
 /docs/resource/host.md    @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
 /model/resource/host.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
 
+# Mobile semantic conventions approvers
```
Comment (reyang) [inline .github/CODEOWNERS:44]: Minor, consider alphabetic ordering.

Comment (breedx-splk) [reply .github/CODEOWNERS:44]: The sections weren't perfectly ordered before (schemas was near the top). I'm happy to do that, but should I move `Schemas` as part of it?

Comment (reyang) [reply .github/CODEOWNERS:44]: I don't have strong opinion so either way should work for me. I would suggest scoping it to mobile semantic conventions only, other parts (that are not perfectly ordered) can be handled in a separate PR.

Comment (breedx-splk) [reply .github/CODEOWNERS:44]: I think the mobile section is already internally sorted alphabetically (`docs` ahead of `mobile`)

**Review by arminru — APPROVED** [review]

**Pull request merged**